python/pyarrow/table.pxi

```diff
@@ -701,18 +701,25 @@ cdef class Table:
         return result
 
     @classmethod
-    def from_pandas(cls, df, bint timestamps_to_ms=False,
+    def from_pandas(cls, df,
+                    object coerce_timestamps=False, object timestamps_to_ms=None,
```

Comment:
note that the signature is slightly changed here; I could add the new argument at the end if you prefer.